test/writer.xwiki

```diff
@@ -623,8 +623,8 @@ or here: <http://example.com/>
 
 From “Voyage dans la Lune” by Georges Melies (1902):
 
-[[image:lalune.jpg||alt="lalune" title="fig:Voyage dans la Lune"]]
-
+(((
+[[image:lalune.jpg||alt="lalune" title="Voyage dans la Lune"]])))
```

Comment:
It's odd that there's a line break after `(((` but not before `)))`, is there a reason for this?

Reply:
I took a look at the [docs](https://www.xwiki.org/xwiki/bin/view/Documentation/UserGuide/Features/XWikiSyntax/?syntax=2.1&section=Groups) and couldn't find a reason, so I've added a newline before the final `)))`.